app/Http/Controllers/Order/InvoiceController.php

```diff
@@ -347,13 +347,15 @@ public function createInvoiceItems($invoiceid, $cart)
     public function invoiceGenerateByForm(Request $request, $user_id = '')
     {
         $this->validate($request, [
+            'user'      => 'required',
             'date'      => 'required|date',
             'domain'    => 'sometimes|nullable|regex:/^(?!:\/\/)(?=.{1,255}$)((.{1,63}\.){1,127}(?![0-9]*$)[a-z0-9-]+\.?)$/i',
             'plan'      => 'required_if:subscription,true',
             'price'     => 'required',
             'product'   => 'required',
         ], [
```

Comment:
If you have more number of rules, better create in Request file Which will look good clean code and better approach